--- gpscorrelate-1.6.1/Makefile.link	2010-04-29 17:20:51.000000000 +0200
+++ gpscorrelate-1.6.1/Makefile	2010-04-29 17:21:42.000000000 +0200
@@ -18,10 +18,10 @@
 all:	gpscorrelate gpscorrelate-gui gpscorrelate.1
 
 gpscorrelate: $(COBJS)
-	g++ $(OFLAGS) -o $@ $(COBJS)
+	g++ -o $@ $(COBJS) $(LDFLAGS) $(OFLAGS)
 
 gpscorrelate-gui: $(GOBJS)
-	g++ $(OFLAGS) -o $@ $(GOBJS)
+	g++ -o $@ $(GOBJS) $(LDFLAGS) $(OFLAGS) $(OFLAGS_GUI)
 
 .c.o:
 	gcc $(CFLAGS) -c -o $*.o $<
